src/tag.c

```diff
@@ -2247,179 +2267,159 @@ find_tags(
 		}
 
 		/*
-		 * Add the found match in ga_match[mtt], avoiding duplicates.
+		 * Add the found match in ht_match[mtt].
 		 * Store the info we need later, which depends on the kind of
 		 * tags we are dealing with.
 		 */
-		if (ga_grow(&ga_match[mtt], 1) == OK)
-		{
-		    int len;
-		    int heuristic;
+		int len;
+		int heuristic;
```

Comment:
Looks the declarations of `len` and `heuristic` need to be moved somewhere (just after the left curly brace at line 2217, maybe?) to comply with ANSI/ISO C90.  Sorry about my being `-pedantic` :)

FWIW, I tried this patch against 7.4.2342 on OS X 10.11.6 (El Capitan) with the same configuration as [the failed CI test](https://travis-ci.org/vim/vim/jobs/158148466); the test fully succeeded for me.